ChatQnA/README.md

```diff
@@ -60,15 +60,18 @@ To set up environment variables for deploying ChatQnA services, follow these ste
 
 3. Set up other environment variables:
 
-   > Notice that you can only choose **one** command below to set up envs according to your hardware. Other that the port numbers may be set incorrectly.
+   > Notice that you can only choose **one** hardware option below to set up envs according to your hardware. Other that the port numbers may be set incorrectly.
```

Comment:
Do you mean:
`Other than` or `Other than that`?

Reply:
I think the correct sentence should be:
`Also make sure port numbers are set correctly too.`